Android/app/src/main/res/layout/fragment_intro.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.motion.widget.MotionLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/intro_motion_layout"
+    android:background="@color/black"
+    app:layoutDescription="@xml/intro_motion_scene">
+
+    <androidx.constraintlayout.utils.widget.ImageFilterView
+        android:id="@+id/hand_watermark"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:src="@drawable/hand_watermark"
+        app:altSrc="@drawable/hand_watermark_red"
```

Comment:
We can remove this line and the resource if you are ok with my other suggestion in the Motion Layout.